frontend/src/api/k8s/configMaps.ts

```diff
@@ -10,6 +10,12 @@ import { ConfigMapModel } from '~/api/models';
 import { genRandomChars } from '~/utilities/string';
 import { applyK8sAPIOptions } from '~/api/apiMergeUtils';
 
+export const CONFIGMAP_PREFIX = 'configmap-';
+
+export const getGeneratedConfigMapName = (): string => `${CONFIGMAP_PREFIX}${genRandomChars()}`;
+export const isGeneratedConfigMapName = (name: string): boolean =>
+  Boolean(name.match(new RegExp(`^${CONFIGMAP_PREFIX}[a-z0-9]{6}$`)));
```

Comment:
Though that might be the same result, just curious, would it be better to use ``new RegExp(`^${CONFIGMAP_PREFIX}[a-z0-9]{6}$`).test(name)`` instead of forcing to cast the type here?

Reply:
yeah its cleaner to use test. i made the switch can you readd the lgtm